lib/data/list_account.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:clone_instagram/constants/source_image.dart';
+import 'package:flutter/material.dart';
+import '../constants/source_string.dart';
+
+class Account {
```

Comment:
fake data should stored primary type or data class

Reply:
i changed